include/common/Driver.hpp

```diff
@@ -687,7 +638,7 @@ class Driver
 	inline cadet::IModelSystem* model() const { return _sim->model(); }
 
 	inline void setWriteLastState(bool writeLastState) CADET_NOEXCEPT { _writeLastState = writeLastState; }
-	inline void setWriteLastStateUnit(bool writeLastStateUnit) CADET_NOEXCEPT { _writeLastStateUnit = writeLastStateUnit; }
+	inline void setWriteLastStateOfUnit(UnitOpIdx uid, bool writeLastStateUnit) CADET_NOEXCEPT {_writeLastStateUnit = writeLastStateUnit; };
```

Comment:
The implementation should add the `uid` to `_writeLastStateUnitId` if `writeLastStateUnit` is `true` and `uid` is not already in `_writeLastStateUnitId`. If `writeLastStateUnit` is `false`, `uid` should be removed from `_writeLastStateUnitId` (if it is in `_writeLastStateUnitId`).